libraries/engage/product/components/UnitQuantityPicker/UnitQuantityPicker.jsx

```diff
@@ -130,7 +150,15 @@ const UnitQuantityPicker = ({
       >
         -
       </RippleButton>
-      <span>
+      <span className={styles.quantityLabelWrapper(inputBgColor)}>
+        <div aria-hidden className={styles.quantityLabel(inputColor)}>
+          {
+            label && label !== ''
+              ? label
```

Comment:
Requirement from ticket:
```
Make prefix wording customizable

    add theme config for a property key
    value of property is used as the prefix
        i.e. "bottles" instead of "quantity"

```

so the config should contain the name of the product property that is used as a label.
So search for a property where its label == LabelFromConfig and use its value in the frontend

Reply:
@Carsten89 this is fixed now, but please test it also for when the `CartUnitQuantityPicker` component is rendered, I wasn't able to reproduce the case.

also, I saw that the component wasn't really centered as asked on the ticket, although the chrome dev tool looked fine, but I guess it would face a problem when the property value gets too long.